### PR TITLE
[Draft] Update display pill

### PR DESCRIPTION
**Pull request by nqhhdev**
https://github.com/linagora/twake-on-matrix/assets/99852347/6865269b-c3b7-4a11-8106-62ad7e55d48d

### CODE REVIEW

**Comment by hoangdat** [comment]
- [ ] pill in chat list
- [ ] pill in search 
- [ ] pill in search inside

File: lib/pages/chat/events/html_message.dart [hunk 1]
```diff
@@ -177,18 +182,25 @@ class HtmlMessage extends StatelessWidget {
...
-          onTap: !room.isDirectChat ? onTap : null,
-          textStyle: !room.isDirectChat
-              ? defaultTextStyle?.copyWith(
-                  color: themeData.colorScheme.primary,
-                )
-              : Theme.of(context).textTheme.bodyLarge?.copyWith(
-                    color: Theme.of(context).colorScheme.onSurface,
-                  ),
+        final displayName =
+            user?.id.substring(0, user.id.indexOf(':')) ?? identifier;
```
Comment (sherlockvn) [inline lib/pages/chat/events/html_message.dart:186]: user.id.indexOf(':') maybe -1, so please handle the error

File: lib/utils/html/html_parser.dart [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      if (node is dom.Element) {
+        final tag = node.localName?.toLowerCase();
+        if (_supportedBlockElements.contains(tag)) {
+          return false;
+        }
+        if (tag == 'a') return true;
+      }
+    }
+    return false;
+  }
```
Comment (hoangdat) [inline lib/utils/html/html_parser.dart:61]: please use `Document.getElementsByTagName()`

File: lib/utils/html/html_parser.dart [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+  static String getContentHasPill(String content) {
```
Comment (hoangdat) [inline lib/utils/html/html_parser.dart:63]: idem

**Pull request closed without merging**